apps/payments/next/tsconfig.storybook.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "emitDecoratorMetadata": true
+  },
+  "exclude": ["components/**/*.spec.ts", "components/**/*.test.ts"],
+  "include": [
+    "components/**/*.stories.ts",
+    "components/**/*.stories.js",
+    "components/**/*.stories.jsx",
+    "components/**/*.stories.tsx",
+    "components/**/*.stories.mdx",
+    ".storybook/*.js",
+    ".storybook/*.ts"
```

Comment:
maybe same comment here re: unused globs.
Reducing glob lookups might shave off 1-3ms from total build time. #terribleperfadvice

Reply:
Lol at the hashtag. You make dev awesome @pdehaan .

Agreed. I included a question about this in the above reply. I'm good with leaving a comment or removing altogether if it's not necessary.